src/workloads/execution/FilterWithLogicalExpressions.yml

```diff
@@ -0,0 +1,375 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
```

Comment:
Should we rename this file `FilterWithComplexLogicalExpression.yml`? That might help us understand what this workload is about in the future, since it's specifically about deep/wide expressions. Another alternative is to call it `FilterWithDeepOrWideLogicalExpression.yml` but that's a bit wordier.